codeception.yml

```diff
@@ -21,4 +21,5 @@ coverage:
     enabled: true
```

Comment:
Please remove `bootstrap: _bootstrap.php` in line 14. It's not necessary anymore while adding property in line 25